lib/Listener/TermsOfServiceEventListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2024 Sagar Gurung <sagar@jankaritech.com>
+ *
+ * @author Sagar Gurung <sagar@jankaritech.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+namespace OCA\OpenProject\Listener;
+
+use OCA\OpenProject\AppInfo\Application;
+use OCA\OpenProject\Service\OpenProjectAPIService;
+use OCA\TermsOfService\Events\SignaturesResetEvent;
+use OCA\TermsOfService\Events\TermsCreatedEvent;
+use OCP\App\Events\AppEnableEvent;
+use OCP\DB\Exception;
```

Comment:
```suggestion
use OCP\DB\Exception as DBException;
```

and then catch DBException in line 76.
I feel like the name `Exception` is dangerous. I also got confused about if you wanted to catch `\Exception` or `OCP\DB\Exception`